po/meson.build

```diff
@@ -1,6 +1,12 @@
 i18n = import('i18n')
 
+run_target(
+  'build',
+  command: './autogen.sh'
+)
+
 i18n.gettext(
   'lutris',
   install_dir: localedir,
+  args: '--from-code=UTF-8'
```

Comment:
This isn't necessary

Reply:
> This isn't necessary

This **is** necessary because there are some Chinese full-width ellipsis in the python code (`…`, `\u2026`). I was surprised by this but it does exist:

![image](https://user-images.githubusercontent.com/26923506/84933620-c15a5980-b108-11ea-9985-e8b4d9564245.png)

Without this you will get some errors:

```
xgettext: Non-ASCII string at /parent/work/winegame/lutris/./lutris/gui/dialogs/runner_install.py:222.
          Please specify the source encoding through --from-code or through a comment
          as specified in http://www.python.org/peps/pep-0263.html.
```

Reply:
That's super strange. Never had that problem with non-ascii characters (like ä, å, etc), but I guess Chinese is a bit different from that.